src/main/java/org/jboss/pnc/logprocessor/eventduration/DateParser.java

```diff
@@ -25,6 +44,7 @@ public static Instant parseTime(String time) {
                 // try next one
             }
         }
+        errCounter.increment();
```

Comment:
Does this mixture on injection and static methods work with Quarkus ?

Reply:
It works as long as we do not use injected field in static method.